gemfire-core/src/main/java/com/gemstone/gemfire/distributed/internal/tcpserver/TcpClient.java

```diff
@@ -121,7 +121,7 @@ public static Object requestToServer(InetAddress addr, int port, Object request,
     sock.setSoTimeout(timeout);
     try {
       DataOutputStream out=new DataOutputStream(sock.getOutputStream());
-      if (serverVersion < Version.CURRENT_ORDINAL) {
+      if (serverVersion < Version.CURRENT_GFE_ORDINAL) {
```

Comment:
This is backward compatible. If version is smaller then create a versioned data stream else a normal one.